verilog/analysis/checkers/line_length_rule.cc

```diff
@@ -70,16 +70,21 @@ const LintRuleDescriptor& LineLengthRule::GetDescriptor() {
       .desc =
           "Checks that all lines do not exceed the maximum allowed "
           "length. ",
-      .param = {{"length", absl::StrCat(kDefaultLineLength),
-                 "Desired line length"}},
+      .param =
+          {
+              {"length", absl::StrCat(kDefaultLineLength),
+               "Desired line length"},
+              {"check_comments", "false", "Check comments."},
```

Comment:
Did we previously check the length in comments ? Then maybe this should default to 'true'.